ads/google/a4a/test/test-utils.js

```diff
@@ -248,5 +262,46 @@ describe('Google A4A utils', () => {
         });
       });
     });
+    it('should specify that this is canary', () => {
+      return createIframePromise().then(fixture => {
+        setupForAdTesting(fixture);
+        const doc = fixture.doc;
+        const elem = createElementWithAttributes(doc, 'amp-a4a', {
+          'type': 'adsense',
+          'width': '320',
+          'height': '50',
+        });
+        const impl = new MockA4AImpl(elem);
+        noopMethods(impl, doc, sandbox);
+        impl.win.AMP_CONFIG = impl.win.AMP_CONFIG || {};
+        impl.win.AMP_CONFIG.canary = true;
+        return fixture.addElement(elem).then(() => {
+          return googleAdUrl(impl, '', 0, [], []).then(url1 => {
+            expect(url1).to.match(/isc=1/);
```

Comment:
This particular param happens to be the first in the query string, so rather than assuming the order in the query string, I'll just use "isc=1", if that's okay.